polymorpheus.gemspec

```diff
@@ -21,5 +21,4 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('rake', '~> 10.4.2')
   s.add_development_dependency('rspec', '~> 2.14.0')
-  s.add_development_dependency('mysql2', '~> 0.3.10')
```

Comment:
So this commit moves the `mysql2` development dependency into version-specific files, and drops the version from 0.3.10 to 0.3.0? Is the latter change intentional?